USDA_data.py

```diff
@@ -1,38 +1,31 @@
-import os
+import matplotlib.pyplot as plt
 import pandas as pd
+import os
+ 
+# Read input data into dataframe df
+df = pd.read_csv('USDA_data.csv', low_memory=False)
```

Comment:
Your merge with `main` undoes a lot of changes from previous commits. Try to reconcile these so that your changes are only for your new feature.